nss-c31-unusual-prestige/nutshell
# My events form

# Description
Created a form so that the user could create new events.  The data passed is pushed to the API. Then, the container that holds the list of events will refresh with the newly created event object.

Applies to issue # 3

## Type of change

Please delete options that are not relevant.



# Testing Instructions
1. Go to ./src/lib and run grunt using the `grunt` command.
2. Open your browser.
3. Open up dev tools and refresh.  You should be met with the current lists of events and a form at the top for adding new events.
4. enter data into the three fields and hit the "Save Event" button.
**Expected result:** You should see the event that you entered into the three fields appear in the list.



# Checklist:
